crates/ide_assists/src/handlers/add_turbo_fish.rs

```diff
@@ -77,7 +77,13 @@ pub(crate) fn add_turbo_fish(acc: &mut Assists, ctx: &AssistContext) -> Option<(
         }
     }
 
-    let number_of_arguments = generics.len();
+    let number_of_arguments = generics
+        .iter()
+        .filter(|param| match param {
+            hir::GenericParam::TypeParam(_) | hir::GenericParam::ConstParam(_) => true,
+            _ => false,
+        })
```

Comment:
```suggestion
        .filter(|param| matches!(param, hir::GenericParam::TypeParam(_) | hir::GenericParam::ConstParam(_)))
```
can be simplified a bit with a special macro exposed by the std prelude(it basically expands to the `match` you have written but is more concise)

Reply:
This is great, thanks! I was hoping there would be a less verbose way of writing this